frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -199,6 +208,50 @@ const FormConceptGroup = (props: Props) => {
             ? t("externalForms.common.concept.copying")
             : props.attributeDropzoneText
         }
+        dropBetween={(i: number) => {
+          return (item: PossibleDroppableObject) => {
+            if (item.type !== DNDType.CONCEPT_TREE_NODE) return;
+
+            if (props.isValidConcept && !props.isValidConcept(item))
+              return null;
+
+            const concept = isMovedObject(item)
+              ? copyConcept(item)
+              : initializeConcept(item, defaults, tableConfig);
+            let newPropsValue = props.value;
+            let insertIndex = i;
+            if (isMovedObject(item)) {
+              const { movedFromFieldName, movedFromAndIdx, movedFromOrIdx } =
+                item.dragContext;
+
+              if (movedFromFieldName === props.fieldName) {
+                if (i > movedFromAndIdx && movedFromOrIdx === 0) {
```

Comment:
Say I have
```
andidx 0 - concept a
andidx 1 - concept b, concept c
andidx 2 - concept d
andidx 3 - concept e
```
and I drag concept `b` down between concepts `d` and `e`, this will decrease the insertIndex and actually insert before concept `d`. That's a bug, right? What is this if condition testing exactly? Maybe it helps to extract this into a variable name?

Reply:
The index of where we want to insert may change if we insert below (the concept is the only one in the andidx). => Will extract and fix this in the next commit